app/pods/search/agenda-items/route.js

```diff
@@ -3,9 +3,9 @@ import Route from '@ember/routing/route';
 import { isEmpty } from '@ember/utils';
 import moment from 'moment';
 import search from 'fe-redpencil/utils/mu-search';
-import DataTableRouteMixin from 'ember-data-table/mixins/route';
+import Snapshot from '../../../utils/snapshot';
```

Comment:
Kleine opmerking.. we gebruiken fe-redpencil ipv terug te gaan in het pad bij impors

Reply:
fixed it